source/common/res/features/toggle-splits/main.js

```diff
@@ -111,7 +111,10 @@
       };
     }());
 
-    ynabToolKit.toggleSplits.invoke();
+    let router = ynabToolKit.shared.containerLookup('router:main');
+    if (router.get('currentPath').indexOf('accounts') > -1) {
+      ynabToolKit.enhancedSelectedTotals.invoke();
```

Comment:
I didn't fix this before we released :( it's not a big deal but I do need to change it to invoke itself lol